koch.nim

```diff
@@ -53,6 +53,10 @@ Options:
   --latest                 bundle the installers with a bleeding edge Nimble
   --stable                 bundle the installers with a stable Nimble (default)
   --nim:path               use specified path for nim binary
+  --localdocs[:path]       only build local documentations. If a path is not
+                           specified (or empty), the default is used.
+                           Additionally options passed to docs will be used to
+                           build local documentation.
```

Comment:
you could add: `eg: --doccmd:skip` (as was written in comments before this PR) since this is an important use case for localdocs

Reply:
I would rather that we detect and strip `googleAnalytics` from the argument list instead, then you can just reuse the rest of the arguments. But that can be done in a future PR.